Containerfile

```diff
@@ -23,7 +23,8 @@ RUN make install-tools && pdm config python.use_venv false && make pdm-lock-chec
 COPY ocp-product-docs-plaintext ./ocp-product-docs-plaintext
 COPY runbooks ./runbooks
 
-COPY scripts/download_embeddings_model.py .
+COPY scripts/download_embeddings_model.py scripts/common_embeddings.py .
+
```

Comment:
To be pedantic, either scripts/common_embeddings.py belongs on line 33 or line 26 needs to subsume line 33.